packages/pds/src/sequencer/index.ts

```diff
@@ -112,16 +112,32 @@ export class Sequencer extends (EventEmitter as new () => SequencerEmitter) {
       .selectFrom(seqQb.as('repo_seq'))
       .innerJoin('repo_blob', (join) =>
         join
-          .onRef('repo_blob.did', '=', 'repo_blob.did')
+          .onRef('repo_blob.did', '=', 'repo_seq.did')
```

Comment:
👀 💎

Reply:
yeah can't believe that one slipped through 😅